ci/testkomodo.sh

```diff
@@ -0,0 +1,14 @@
+
+copy_test_files () {
+    cp -r $CI_SOURCE_ROOT/tests $CI_TEST_ROOT/tests
+    cp $CI_SOURCE_ROOT/pyproject.toml $CI_TEST_ROOT
+}
+
+install_test_dependencies () {
+    pip install -e ".[ert]"
```

Comment:
This line is not needed. Both completor and ert are already installed in the Komodo environment where this is run.